webapp/src/Command/ImportEventFeedCommand.php

```diff
@@ -1979,7 +1979,7 @@ protected function processPendingEvents(string $type, $id)
             // Get all pending events
             $pending = $this->pendingEvents[$type][$id];
             // Mark them as non-pending. Note that they might depend on more events,
-            // but then they'll be readded automatically in the correct place
+            // but then they'll be read automatically in the correct place
```

Comment:
This *was* correct.

Reply:
Fixed in my other PR as re-added. I wrongly assume `read` as in reading something into an array.